module/zfs/dbuf.c

```diff
@@ -5117,3 +5141,6 @@ ZFS_MODULE_PARAM(zfs_dbuf, dbuf_, cache_shift, INT, ZMOD_RW,
 
 ZFS_MODULE_PARAM(zfs_dbuf, dbuf_, metadata_cache_shift, INT, ZMOD_RW,
 	"Set size of dbuf metadata cache to log2 fraction of arc size.");
+
+ZFS_MODULE_PARAM(zfs_dbuf, dbuf_, mutex_hash_shift, UINT, ZMOD_RD,
```

Comment:
Should this be `ZMOD_RW`?

Reply:
Since the hash is initialized at module load time there's no need to make it writable.  This value can still be set with a module option when loading the kmod.